Scripts/script-GetTime.yml

```diff
@@ -4,8 +4,14 @@ commonfields:
 name: GetTime
 script: |-
   var d = new Date();
-  if ((args.date) && (args.date.length > 0)){
-      d =  new Date(args.date);
+  if (args.date){
+      if (args.date.length > 0){
+          // args.date is string
+          d =  new Date(args.date);
+      } else {
+          // args.date is object (Time.time in Go)
```

Comment:
can we check if its actually an object, and else throw exception?

Reply:
we can but it wouldn't really provide any value since this arg can only be used as string or object, anything else would fail anyway. Even if it's object but not Time.time in Go.

Reply:
And there is no actual way for me to determine that this object is Go's time.time object

Reply:
ok leave as is and lets just add test for the case is `Time.time`